app/services/proofing/mock/resolution_mock_client.rb

```diff
@@ -14,6 +14,7 @@ def proof(applicant)
         return failed_to_contact_vendor_result if /Fail/i.match?(first_name)
         return failed_to_contact_vendor_result if ssn.match?(NO_CONTACT_SSN)
         return timeout_result if first_name.match?(/Time/i)
+        return parse_error_result if first_name.match?(/Parse/i)
```

Comment:
looks like we have a lot of checks on `first_name` and they're in different orders, what if we reorganized this a tad?

```suggestion
        return failed_to_contact_vendor_result if ssn.match?(NO_CONTACT_SSN)
        case first_name
        when /Fail/i
          return failed_to_contact_vendor_result
        when /Time/i
          return timeout_result
        when /Parse/i
          return parse_error_result
        end
```

or if we want to keep it vertically more "snug" we coudl use `when...then`

looks like we have a lot of checks on `first_name` and they're in different orders, what if we reorganized this a tad?

```suggestion
        return failed_to_contact_vendor_result if ssn.match?(NO_CONTACT_SSN)
        case first_name
        when /Fail/i then return failed_to_contact_vendor_result
        when /Time/i then return timeout_result
        when /Parse/i then return parse_error_result
        end
```